fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -832,7 +832,7 @@ public class Config extends ConfigBase {
      * Parallel load fragment instance num in single host
      */
     @ConfField(mutable = true)
-    public static int load_parallel_instance_num = 1;
+    public static int load_parallel_instance_num = Runtime.getRuntime().availableProcessors() / 2;
```

Comment:
Sometimes, the CPU cores between FE and BE are not the same. FE needs fewer resources and has fewer CPU cores. So I think we can use the BE cores to set the variable. `getAvgNumOfHardwareCoresOfBe() in system/BackendCoreStat.java`

Reply:
thank you, @chaoyli 
get comments addressed